en/developers-guide/proof-of-reserve.md

```diff
@@ -0,0 +1,47 @@
+---
+description: Ongoing proof of reserve validation for your financial entity.
+---
+
+# Proof of Reserve
+
+For a comprehensive illustration of utilizing **Orakl Network's Proof of Reserve**, refer to the 
+example repository of the [`data-feed-consumer`](https://github.com/Bisonai/data-feed-consumer). The on-chain implementation for Proof of Reserve mirrors that of the Aggregator on the Aggregator Proxy.
+
+## What is Proof of Reserve?
+
+The **Orakl Network's Proof of Reserve (PoR)** stands as a vital element in fostering trust and transparency within the financial ecosystem of 
+Orakl Network. Playing a pivotal role, Proof of Reserve is dedicated to the verification and validation of reserve holdings for financial 
+entities through a secure and auditable procedure.The Peg-Gold Proof of Reserve refers to a reserved quantity of non-fungible tokens (NFTs) issued specifically for PEG Gold.
+
+
+### Supported Proof of Reserve on Cypress
+
+<table><thead><tr><th width="157">PoR</th><th>Aggregator</th><th>AggregatorProxy</th><th>Heartbeat (m)</th></tr></thead><tbody>
+    <tr>
+        <td>PEG-POR</td>
```

Comment:
How about calling it as `GPC (Gold Pegged Coin)` as in https://orakl.network/proof-of-reserve?